docs/content/en/docs/install/k8s.md

```diff
@@ -132,6 +132,38 @@ Some considerations for Keptn:
   So you can create `KeptnMetrics` in a centralized namespace
   (such as `keptn-lifecycle-toolkit`)
   and access those metrics in evaluations on all namespaces in the cluster.
+* The resources used for analyses
+  ([Analysis](../yaml-crd-ref/analysis.md),
+  [AnalysisDefinition](../yaml-crd-ref/analysisdefinition.md),
+  and
+  [AnalysisValueTemplate](../yaml-crd-ref/analysisvaluetemplate.md))
+  each support an optional `namespace` field.
```

Comment:
the namespace is not clear here at all, what namespace?

Reply:
It's a field in the resource so detailed if you look at the referenced page.  This is a high-level explanation so not sure what sort of details you expect.  We're just saying that you can specify a namespace.